api/user_models.py

```diff
@@ -0,0 +1,54 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+#
+# Copyright (C) 2023 Collabora Limited
+# Author: Jeny Sadadia <jeny.sadadia@collabora.com>
+
+# Disable flag as user models don't require any public methods
+# at the moment
+# pylint: disable=too-few-public-methods
+
+"""User model definitions"""
```

Comment:
It might be worth mentioning `fastapi-users` here to say this is the "glue" code for our particular API application.